core-ui/src/sidebar/Footer/useGetLegalLinks.ts

```diff
@@ -0,0 +1,31 @@
+import { useTranslation } from 'react-i18next';
+import { useMicrofrontendContext } from 'shared/contexts/MicrofrontendContext';
+
+type LegalLink = {
+  label: string;
+  link: string;
+};
+
+export const useGetLegalLinks = (): LegalLink[] => {
+  const { t, i18n } = useTranslation();
+  const { features } = useMicrofrontendContext() as any;
```

Comment:
So, as the microfrontendContext is going to be dropped, the new components should subscribe to the recoil store. There is already a recoil atom for features `configFeaturesAtom` . Actually maybe I'll rewrite the features state today